packages/config/src/layer2s/linea.ts

```diff
@@ -50,6 +48,17 @@ const roles = discovery.getContractValue<{
   PAUSE_MANAGER_ROLE: { members: string[] }
 }>('zkEVM', 'accessControl')
 
+const zodiacRoles = discovery.getContractValue<{
+  roles: Record<string, Record<string, boolean>>
+}>('Roles', 'roles')
+const zodiacPauserRole = '1'
+const zodiacPausers: ScalingProjectPermissionedAccount[] = Object.keys(
+  zodiacRoles.roles[zodiacPauserRole].members,
+).map((zodiacPauser) => ({
+  address: EthereumAddress(zodiacPauser),
+  type: 'EOA',
+}))
```

Comment:
Please use `discovery.formatPermissionedAccount` or assert that those pausers are indeed eoas here.

Reply:
[chore: use discovery formatter for zodiac pausers](https://github.com/l2beat/l2beat/pull/2565/commits/4cb77c1eccabd04629aa6aadf70d7b0127bbca02)